microsoft-edge/devtools-guide-chromium/remote-debugging/local-server.md

```diff
@@ -113,7 +113,8 @@ If your web server is running off of a non-standard port, remember to specify th
 
 [CharlesWebDebuggingProxy]: https://www.charlesproxy.com "Charles Web Debugging Proxy"  
 
-[SquidOptimisingWebDelivery]: https://www.squid-cache.org "squid : Optimising Web Delivery"  
+[SquidOptimisingWebDelivery]: http://www.squid-cache.org "squid : Optimising Web Delivery"  
+<!-- this link must be http, not https -->  
```

Comment:
This link seems better anyways: https://wiki.squid-cache.org/ so I think we can just replace this one